runtime/doc/filetype.txt

```diff
@@ -513,40 +513,40 @@ Local mappings:
 MAN					*ft-man-plugin* *:Man* *man.vim*
 
 View manpages in Nvim. Supports highlighting, autocompletion, locales, and
-navigation. See also |find-manpage|.
+navigation. Also see |find-manpage|.
 
 To use Nvim as a manpager:
 
      export MANPAGER="nvim -c 'set ft=man' -"
 
+All commands and mappings will attempt to reuse the closest man window
+(left/right) and otherwise create a horizontal split.
+
 Commands:
```

Comment:
Guess I'd expect the tag for the command to bring you to the help for the command, not to the top of the file containing the command.